README.md

```diff
@@ -29,3 +29,10 @@ This is usable as is in v0.2.x but in v0.3.0 and above, you need to add a file t
 * `-Dpekko.build.pekko.version=main` will find latest snapshot for `main` branch (December 2023 - this evals to `1.1.0-M0+...-SNAPSHOT`)
 * `-Dpekko.build.pekko.version=1.0.x` will find latest snapshot for `1.0.x` branch
 
+## PekkoDevelocityPlugin
+
+This autoplugin enables automatic publishing of Build Scan® for every CI build, as well as for every local build executed by an authenticated Apache committer, to the [Apache Develocity](https://ge.apache.org) instance.
```

Comment:
```suggestion
This autoplugin enables automatic publishing of [Develocity](https://gradle.com/) Build Scan® for every CI build, as well as for every local build executed by an authenticated Apache committer, to the [Apache Develocity](https://ge.apache.org) instance.
```

Reply:
Can we change this to only be CI builds? I am against forcing developers to submit results to Gradle. For me, it is against ASF principles to take results from user machines without their explicit opt-in.

I guess I am on the fence about plugging in proprietary build tooling. I have never used Develocity and have very little interest in using it. I actually feel a bit put out by the idea that I have to now use it to test this out.

Is there some kind of documentation about what info the tool ingests? Is there a contact in the ASF that has been involved with setting up Develocity who I can get some ASF tinged advice on whether it is a good idea for us to engage with this?

If Develocity causes us any issues, I will very quickly seek to remove it from our builds.